src/com/andrewkingmarshall/Utils/PlaylistHandler.java

```diff
@@ -45,12 +45,21 @@ public void joinPlaylist(SpoqUser spoqUser) throws PlaylistNotFoundException{
         if(playlistExists(spoqUser.getConnectedPlaylistId())) {
             spoqUser.setAdmin(false);
             playlistMap.get(spoqUser.getConnectedPlaylistId()).addUser(spoqUser);
+        }else{
+            throw new PlaylistNotFoundException();
         }
     }
 
     public void leavePlaylist(SpoqUser spoqUser) throws PlaylistNotFoundException{
-        if(playlistExists(spoqUser.getConnectedPlaylistId())) {
-            playlistMap.get(spoqUser.getConnectedPlaylistId()).removeUser(spoqUser.getMusicServiceId());
+        int playlistId = spoqUser.getConnectedPlaylistId();
+        System.out.println("Leaving playlist: " + playlistId);
```

Comment:
If we have a printLn saying we are leaving the playlist and then we don't because the playlist is not there it may be confusing to debug. What do you think about putting a log print statement in an Else saying, Playlist not found?